src/extension.ts

```diff
@@ -228,6 +275,11 @@ function fetchAndLaunchMetals(context: ExtensionContext, javaHome: string) {
   );
 }
 
+function updateJavaConfig(javaHome: string, global: boolean = true) {
+  workspace.getConfiguration("metals").update("javaHome", javaHome, global);
+  commands.executeCommand("workbench.action.reloadWindow");
```

Comment:
Let's not reload automatically, Metals will already prompt the user to reload the window when settings change. Some users may have state in their terminal that they don't want to loose

Reply:
please note that there's a generalized way to do this in response to configuration changes. See https://github.com/scalameta/metals-vscode/blob/136ef625062b690e9ac22a9464c8ac8d281a7d1a/src/extension.ts#L729-L754

Reply:
(actually, `javaHome` is already in there, so the user should be prompted automatically already after the `update`)

Reply:
I've removed auto-reload.